### DISCUSSION TITLE
ClashMetaForAndroid不支持PROCESS-NAME规则，clash for Android则支持。

### DISCUSSION THREAD

**Discussion by huangdog** [MetaCubeX/ClashMetaForAndroid#400] [General]
ClashMetaForAndroid      ，PROCESS-NAME规则没有生效，会直接被clash meta 忽略，然后匹配到MATCH那里。
find-process-mode: strict
enable-process: true
即使我加了这2个参数，也没有用。

原版clash for Android是支持的。
PROCESS-NAME在Android上是非常好用的，可以以app为单位用规则分流。

**Comment by yehudi-dev** [comment]
according to this: https://github.com/MetaCubeX/mihomo/issues/91 
You might try to use the `uid` of process alternatively.

**Reply by yehudi-dev** [reply]
@huangdog I also have this requirement and would greatly appreciate any insights. If you find a workaround (even if it's not using `Clash Meta for Android` but another `clash` solution), please share your findings!

**Comment by yehudi-dev** [comment]
from : https://github.com/chen08209/FlClash/issues/139
FlClash supports this by `enable-process: true`
